Melt/chain.py

```diff
@@ -107,17 +37,7 @@ def main():
         ci = cacophony_index.calculate(argv[2])
         summary.update(ci)
     elif argv[1] == '-examine':
```

Comment:
if we arenot going to have any other argument options lets just remove the need for -examine

Reply:
okay